docs/v3_services.rst

```diff
@@ -0,0 +1,249 @@
+Securedrop V3 Onion Services
+============================
+
+SecureDrop uses onion services for the *Journalist* and *Source Interfaces*, as
+well as for SSH access in SSH-over-Tor mode. Previous to the
+release of SecureDrop 1.0.0, only v2 onion services were available, but 
+administrators now have the option of enabling next-generation (v3) onion 
+services.
+
+V3 onion services use more secure cryptography than v2 onion services,  and 
+include redesigned protocols that guard against service 
+information leaks on the Tor network. The unique identifier in V3 onion 
+addresses is 56 characters long - for example:
+
+.. code-block:: none
+
+  http://vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion
+
+This makes them easily distinguishable from 16-character v2 onion addresses,
+such as:
+
+.. code-block:: none
+
+  https://www.nytimes3xbfgragh.onion/
+
+For new SecureDrop instances, it is recommended to
+enable v3 onion services only. For existing instances, migration
+to v3 onion services is recommended, and the ability to run both
+v2 and v3 onion services simultaneously is supported to make the process
+easier.
+
+Migrating from v2 to v3 only or v2+v3
+-------------------------------------
+
+Preparing for the migration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before starting the migration process, you should decide whether to move 
+straight to v3 only, or move to v2+v3 temporarily. As the URLs of your onion
+services will change with the move to v3, moving to v2+v3 first will allow
+you to minimize the impact of the migration on sources and journalists.
+
+URL changes will affect the following:
+
+- The *Source Interface* address will change - once the migration is complete, 
+  you will need to update your landing page and other resources that reference
+  the address, such as your SecureDrop directory entry.
+- If your instance uses HTTPS, you will need to provision a new certificate for
+  the v3 *Source Interface* address - this will need to be done `after` the new
+  address has been generated.
+- *Journalist* and *Admin Workstations* will need to be updated to use the v3
+  addresses of the *Journalist* and *Source Interface*, and the SSH proxy 
+  services if appropriate.
```

Comment:
`if appropriate` -> `if your instance is using SSH over Tor`

Reply:
:white_check_mark: